### PR TITLE
Add all grayscale version of badges, change SvgContainer to accomodate for the addition of grayscale badges

### PR DESCRIPTION
**Pull request by thai-truong**
### Administrative Info
Monday Board ID: 853606148
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Add grayscale versions of badge icons so we can show badges obtained/not obtained to users

### Testing
How did you confirm your changes worked? 
- Just adding SVG strings so don't think I needed this

### Confirmation of Change 
Go to src/assets, changes are in the folder badgeXMLs and titletoXMLs, as well as src/components/SvgComponent.

### CODE REVIEW

**Review by bobbytatum27 — APPROVED** [review]
lgtm! thanks for updating the container + adding logic to resolve the xml to either grayscale or colored.

File: src/assets/badges/svgXMLs/dryers.ts [hunk 1]
```diff
@@ -11,4 +11,175 @@ const dryersXML = `<svg width="76" height="70" viewBox="0 0 76 70" fill="none" x
...
+const grayDryersXML = `<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="76"
```
Comment (kunalb123) [inline src/assets/badges/svgXMLs/dryers.ts]: should the width and height on all of these be different bc I think for some of them they're the same

Comment (thai-truong) [reply src/assets/badges/svgXMLs/dryers.ts]: Yes, as the dimensions are obtained from the figma. I don't think the tool I used to grayscale these (Inkscape) modifies height and width of the SVG, so I think this just came from figma itself.

**Review by kunalb123 — CHANGES_REQUESTED** [review]
lgtm; just take a look at the width/height thing

**Comment by thai-truong** [comment]
> 
> 
> lgtm; just take a look at the width/height thing

I think this should be handled by the designers and not me, since I got those badge XMLs wholesale from the Figma itself (and not sure what happens if I change it directly in the XML instead of through figma and then recopy, don't wanna risk ruining the badges).

**Comment by bobbytatum27** [comment]
> > lgtm; just take a look at the width/height thing
> 
> I think this should be handled by the designers and not me, since I got those badge XMLs wholesale from the Figma itself (and not sure what happens if I change it directly in the XML instead of through figma and then recopy, don't wanna risk ruining the badges).

Good catch @kunalb123 ! I'm not too sure what happened on Figma, but I just went in and the dryer badge dimensions were not 70x70. Maybe I mistakenly edited it when fixing the badge dims for Upo the other week? Regardless, @thaigillespie I've gone ahead and made sure all badges on Figma are 70x70. I went thru each diff again and I think this is only an issue for the dryer and ceiling fan (disregard this one: after resolving the issue in [this thread](https://tritonse.slack.com/archives/C01EG0YCWT0/p1620339987257400), the dimensions were still off after exporting).

File: src/assets/badges/svgXMLs/ceilingFans.ts [hunk 2]
```diff
@@ -1,20 +1,268 @@
...
-<path d="M18.1927 43.6533C21.5824 38.0313 28.8878 33.8423 37.8295 34.8344C29.8229 30.8108 18.8941 31.5273 10.6536 37.921C7.96524 39.9604 8.49122 43.984 11.5303 45.4722C11.5887 45.4722 11.5887 45.5273 11.6471 45.5273C13.9849 46.6848 16.8486 45.8029 18.1927 43.6533Z" fill="#2B3B4E"/>
-<path d="M55.3623 47.0153C48.466 46.7949 41.1022 42.7712 37.771 34.8342C37.8294 43.3775 43.6153 52.1413 53.4921 55.9996C56.7065 57.2673 60.1546 54.9523 60.1546 51.7004C60.1546 51.6452 60.1546 51.6452 60.1546 51.5901C60.1546 49.1098 57.9922 47.0704 55.3623 47.0153Z" fill="#2B3B4E"/>
-<path d="M37.8291 40.8979C41.3796 40.8979 44.2578 38.1835 44.2578 34.835C44.2578 31.4865 41.3796 28.772 37.8291 28.772C34.2786 28.772 31.4004 31.4865 31.4004 34.835C31.4004 38.1835 34.2786 40.8979 37.8291 40.8979Z" fill="white"/>
-<path d="M37.8293 38.8583C40.1855 38.8583 42.0956 37.0568 42.0956 34.8347C42.0956 32.6125 40.1855 30.811 37.8293 30.811C35.4731 30.811 33.563 32.6125 33.563 34.8347C33.563 37.0568 35.4731 38.8583 37.8293 38.8583Z" fill="#E6E9EE"/>
-</svg>
-`;
+const ceilingFansXML = `<svg width="71" height="70" viewBox="0 0 71 70" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (kunalb123) [inline src/assets/badges/svgXMLs/ceilingFans.ts:1]: some of these aren't 70x70 still but if it's coming off the figma then I think it's fine

**Review by kunalb123 — CHANGES_REQUESTED** [review]
The sizing is still not square on some of the badges. If this doesn't have a real impact on the actual product then lmk and i'll approve the changes

File: src/assets/badges/svgXMLs/ceilingFans.ts (hunk 2 above)
Comment (thai-truong) [reply src/assets/badges/svgXMLs/ceilingFans.ts:1]: Yes it's coming off of the Figma. Bobby even made everything 70x70 on Figma, but then when I copy the XML strings of the SVG badges it gives 71x70. Not sure why

Comment (kunalb123) [reply src/assets/badges/svgXMLs/ceilingFans.ts:1]: ok i think its good then ill approve it

**Review by kunalb123 — APPROVED** [review]

**Pull request merged**